### ISSUE TITLE
Job summary lists unwanted results for builds invoked by tooling and/or tests

### ISSUE THREAD

**Issue by snazy** [gradle/actions#28]
We have CI workflows/jobs setup w/ the Gradle build action - it's running fine w/o any issues for quite a while now.

The only thing that I do not fully understand is that the build summary lists a lot of "phantom" Gradle builds that requested the `classes` task (see screenshot), although the workflow does not do that, neither do our build files directly.

We do use the Quarkus plugin in our builds, and I suspect that the Quarkus plugin somehow triggers/causes these "phantom" builds listed in the summary.

Would it be possible to hide these "phantom" builds in the action?

![image](https://user-images.githubusercontent.com/957468/207866643-f734b0bc-f29c-4a54-bc8c-930d89b93ffb.png)
[Workflow reference](https://github.com/projectnessie/nessie/blob/main/.github/workflows/pull-request.yml#L65-L76)
"Heads-up" on the Quarkus side: https://github.com/quarkusio/quarkus/issues/29890

**Comment by nnellanspdl** [comment]
I'm having this issue, but I think my problem is slightly different.

I'm one of the strange ones who uses long-lived github runners.  Over time, these runners run many different gradle builds from many different projects.

It has gotten to the point now when I do a single gradle build of a single project, my build summary shows 100's of lines, showing various projects that are not related to my current build.

Is this something to do with cache, or possibly with a cluttered gradle user home?  I've added a step at the beginning of my workflow that clears `~/.gradle` but that doesn't seem to help.

**Comment by bigdaz** [comment]
@nnellanspdl Yes this likely has a different cause. Can you please open a separate issue to track this?

**Comment by bigdaz** [comment]
Gradle executions will not be included in the Job Summary if the `GRADLE_ACTIONS_SKIP_BUILD_RESULT_CAPTURE` environment variable is set.
